### PR TITLE
Support `NO_COLOR` env

### PR DESCRIPTION
**Pull request by Reokodoku**
For more information go [here](https://no-color.org/).

### CODE REVIEW

**Comment by Arnau478** [comment]
Wow, thank you! First contributor! It actually makes sense. I am going to review it and merge it.

File: src/main.zig [hunk 1]
```diff
@@ -116,15 +116,26 @@ pub fn main() !void {
...
 
     const stdout = std.io.getStdOut();
 
+    var color = stdout.supportsAnsiEscapeCodes();
+    if (parsed_args.color) |val| {
+        color = val;
+    } else if (envs.get("NO_COLOR")) |val| {
+        if (!std.mem.eql(u8, val, ""))
+            color = false;
+    }
```
Comment (Arnau478) [inline src/main.zig:135]: It makes sense to have a separate variable for `color`, it looks much cleaner. However, maybe we should avoid using `var`, as then `color` could be modified later on. What do you think of having two chained `orelse`s? Or maybe a separate function, something like `useColor()`? This is not a request for you to change it, I'm just giving you some ideas. If you want to change it, go ahead. If not, tell me and I'll merge it anyway :)

**Review by Arnau478 — CHANGES_REQUESTED** [review]
If you want it that way, don't worry, just tell me and I'll merge it anyway :)

File: src/main.zig (hunk 1 above)
Comment (Reokodoku) [reply src/main.zig:135]: Thanks for the suggestions! I think using a function is better because it is a cleaner solution. I will implement it!

**Review by Arnau478 — APPROVED** [review]
LGTM

**Pull request merged**